### DISCUSSION TITLE
Nvidia Xavier NX & I Robot Create3 & RPlidar A1 & Stereolabs ZED2 camera

### DISCUSSION THREAD

**Discussion by valdivj** [iRobotEducation/create3_docs#161] [Show and tell]
Here is a link to a video that shows a I Robot Create3 being controlled by a Nvidia Xavier NX While also running a object recognition and tracking program  using the stereolabs ZED2 camera and also while showing the output of a RPlidar A1 in RVIZ

https://youtu.be/IKj2wN9y9xY

**Comment by justinIRBT** [comment]
Nice!  If you decrease acceleration limits as mentioned in the Acceleration Limits section of this page https://iroboteducation.github.io/create3_docs/api/safety/ the robot might be a little less jerky with the high CG.  You can set that property through a param set or at application startup through the webserver application configuration if you want to play with it.

**Reply by valdivj** [reply]
Thanks
For the info. 
I will try that

**Comment by shamlian** [comment]
This looks awesome! Thank you for sharing the project. Along with the idea to reduce acceleration, another thought might be to add a felt pad in the rear of the robot, or some other slide or caster, in order to reduce rocking. If you have a 3D printer available, we have a design posted [here](https://iroboteducation.github.io/create3_docs/hw/mechanical/#adding-a-rear-caster).

**Comment by valdivj** [comment]
Wow 
There are all kinds of cool little brackets and mounts in that file.
